packaging/docker/run.sh

```diff
@@ -40,6 +40,13 @@ if [ -n "${PGID}" ]; then
   usermod -a -G "${PGID}" "${DOCKER_USR}" || echo >&2 "Could not add netdata user to group docker with ID ${PGID}"
 fi
 
+if mountpoint -q /etc/netdata; then
+  if [ -n "${NETDATA_COPY_STOCK_CONIG}" ] || [ "$(find /etc/netdata -type f | wc -l)" -eq 0 ]; then
```

Comment:
```suggestion
  if [ -n "${NETDATA_COPY_STOCK_CONIG}" ] && [ "$(find /etc/netdata -type f | wc -l)" -eq 0 ]; then
```

I think we need to do it only once, otherwise we overite the files on every restart.

Reply:
And perhaps "if the dir is empty" is better than "if there are no files in the dir".

`[ -z "$(ls -A /etc/netdata)" ]`